src/main/webapp/app/modules/provider/record/single-record-view.scss

```diff
@@ -27,7 +27,8 @@
     height: auto !important;
   }
 
-  .detail-card {
+  .loc-card,
+  .srv-card {
```

Comment:
I wonder if loc-card and srv-card is repeated below

Reply:
No it's not, there is .srv-card > *, which is not the same as .srv-card